### ISSUE TITLE
Issue with python 3.8

### ISSUE THREAD

**Issue by EspenBrun** [keleshev/value#2]
Hello, I tried to use your package and ran into problem. Maybe due do python version? I use Value version 0.1.0.

Any plans to support 3.8? Seemed like a neat little module.

```python
Python 3.8.5 (tags/v3.8.5:580fbb0, Jul 20 2020, 15:57:54) [MSC v.1924 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from value import Value
>>>
>>> class Date(Value):
...
...     def __init__(self, year, month=1, day=1):
...             pass
...
>>> Date(2013, 3).year
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\EspenKirkesætherBrun\code\excel-templates\venv\lib\site-packages\value.py", line 10, in __new__
    self = object.__new__(class_, *args, **kwargs)
TypeError: object.__new__() takes exactly one argument (the type to instantiate)
>>>
```

**Comment by keleshev** [comment]
Check out `dataclasses` module which is available from Python 3.7, I believe, which supersedes the functionality in this module.

**Comment by EspenBrun** [comment]
I didn't notice that this was a quite old repo when I reported the issue. Thanks for replying and pointing me in the right dirrection :)
